packages/meditrak-server/src/routes/mapOverlays/assertMapOverlaysPermissions.js

```diff
@@ -30,6 +35,44 @@ export const assertMapOverlaysPermissions = async (accessPolicy, models, mapOver
   throw new Error('You do not have permissions for the requested map overlay(s)');
 };
 
+export const assertMapOverlaysEditPermissions = async (accessPolicy, models, mapOverlayId) => {
+  const mapOverlay = await models.mapOverlay.findById(mapOverlayId);
+  if (!mapOverlay) {
+    throw new Error(`No map overlay exists with id ${mapOverlayId}`);
+  }
+
+  const countryCodes = mapOverlay.countryCodes || [];
```

Comment:
I don't think we ever have a mapOverlay without any country codes, so it should be safe to just let it be 'mapOverlay.countryCodes'